etc/portage/package.env

```diff
@@ -1,3 +1,2 @@
 sys-apps/archspec test.conf
-sys-cluster/lmod test.conf
```

Comment:
@trz42 Do you have more information on how the Lmod tests were failing?
Can you open an issue with more info, so we can check the details and see if we can re-enable the tests?

Reply:
Yep. Discussed on Slack Oct 14, 2022. Tests that failed with version 8.6.14
```
diff     R  2.64   rt/avail/avail/t1
diff     R  2.08   rt/changeMPATH/changeMPATH/t1
diff     R  4.19   rt/columnT/columnT/t1
diff     R  0.335  rt/configDir/configDir/t1
diff     R  1.37   rt/csh_swap/csh_swap/t1
diff     R  0.575  rt/deprecation/deprecation/t1
diff     R  1.32   rt/eb/eb/t1
diff     R  0.679  rt/family/family/t1
diff     R  0.247  rt/fish/fish/t1
diff     R  0.302  rt/hook/hook/t1
diff     R  0.963  rt/hook_avail/hook_avail/t1
diff     R  1.11   rt/inherit/inherit/t1
diff     R  0.647  rt/isloaded/isloaded/t1
diff     R  0.575  rt/junk/junk/t1
diff     R  1.35   rt/lars/lars/t1
diff     R  0.383  rt/lastName/lastName/t1
diff     R  8.89   rt/load/load/t1
diff     R  0.741  rt/mpath/mpath/t1
diff     R  0.598  rt/noTCL/noTCL/t1
diff     R  0.407  rt/personal_mf/personal_mf/t1
diff     R  0.621  rt/petsc/petsc/t1
diff     R  0.741  rt/pkg/pkg/t1
diff     R  0.781  rt/purge/purge/t1
diff     R  1.17   rt/reset/reset/t1
diff     R  0.344  rt/rstats/rstats/t1
diff     R  6.74   rt/saverestore/saverestore/t1
diff     R  1.3    rt/sd_tacc/sd_tacc/t1
diff     R  1.83   rt/sh_to_modulefile/sh_to_modulefile/t1
diff     R  0.29   rt/slash/slash/t1
diff     R  3.57   rt/swap/swap/t1
diff     R  2.41   rt/tabcompl/tabcompl/t1
diff     R  1.86   rt/tacc/tacc/t1
diff     R  0.7    rt/uitSitePkg/uitSitePkg/t1
diff     R  0.624  rt/user_inherit/user_inherit/t1

Reply:
Probably similar / related to the issues I experienced as well, see https://github.com/EESSI/compatibility-layer/issues/153.